src/utils/input.ts

```diff
@@ -1,6 +1,8 @@
 import capitalize from 'lodash/capitalize';
 import { RadioItem } from 'src/components/dls/Forms/RadioGroup/RadioGroup';
 import { SelectOption } from 'src/components/dls/Forms/Select';
+import AvailableTranslation from 'types/AvailableTranslation';
```

Comment:
code looks good to me overall mA.

btw what is the difference between `AvailableTranslation` and `Translation`? looks like we have both in `types/`

Reply:
`AvailableTranslation` is the list of all supported translations. `Translation` refers to a verse/word translation.

Reply:
I see.

out of the scope of this PR.
- shall we make “Translation” more specific? Like VerseTranslation?
- Or is it possible to merge them together, not sure how the structure of the backend api now.


IMO there’s a room for improvement in these naming